.unleash/flags.json

```diff
@@ -1,6 +1,22 @@
 {
     "version": 1,
     "features": [
+        {
+            "name": "cost-management.backend.disable-ddf-tag-form",
```

Comment:
I wouldn't even bother putting this flag here. Aren't we only using this flag once in prod?

Reply:
Yeah, I have this flag and the cost categories flag as part of the deprecated clean up issues. So it will be removed relatively soon.